acceptance/openstack/autoscaling/v1/configurations_test.go

```diff
@@ -36,11 +36,12 @@ func TestConfigurationsLifecycle(t *testing.T) {
 	}
 
 	secGroupID := openstack.CreateSecurityGroup(t)
-	defer openstack.DeleteSecurityGroup(t, secGroupID)
+	t.Cleanup(func() {
```

Comment:
This test fails with:
=== RUN   TestConfigurationsLifecycle
    common.go:64: Security group 35507503-24d2-4cf2-8bd2-4eaec85224d6 was created
    configurations_test.go:43: Attempting to create AutoScaling Configuration
    configurations_test.go:73: Failure in configurations_test.go, line 73: unexpected error "Bad request with: [POST https://as.eu-de.otc.t-systems.com/autoscaling-api/v1/5045c215010c440d91b2f7dce1f3753b/scaling_configuration], error message: {\"error\":{\"code\":\"AS.1041\",\"message\":\"The eip info of scaling config is null.\"}}"
    common.go:75: Security group 35507503-24d2-4cf2-8bd2-4eaec85224d6 was deleted
--- FAIL: TestConfigurationsLifecycle (39.58s)

FAIL